website/docs/api/admin/feature-toggles-api.md

```diff
@@ -313,7 +370,11 @@ None
 }
 ```
 
-### Mark a Feature Toggle as "active" {#mark-a-feature-toggle-as-active}
+## Mark a Feature Toggle as "active" {#mark-a-feature-toggle-as-active}
+
+:::caution Deprecation notice
+This endpoint is deprecated. Please use the [project-based endpoint to patch a feature toggle and mark it as not stale](./feature-toggles-api-v2.md#patch-toggle) instead.
```

Comment:
Is this the right place to go for updating staleness?

Reply:
Yes, that's what we do in the GUI at least